gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,7 +1,7 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-# https://gradle.org/release-checksums/
-distributionSha256Sum=e5444a57cda4a95f90b0c9446a9e1b47d3d7f69057765bfb54bd4f482542d548
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.1-bin.zip
+#distributionSha256Sum=29e49b10984e585d8118b7d0bc452f944e386458df27371b49b4ac1dec4b7fda
+#distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.2-bin.zip
+distributionUrl=https://services.gradle.org/distributions-snapshots/gradle-7.5-20220427150934+0000-bin.zip
```

Comment:
I gave 7.5-SNAPSHOT a try, but it still fails with a toolchain and JDK 18:
https://github.com/spockframework/spock/runs/6201180386?check_suite_focus=true#step:4:49

@leonard84 Do you recognize any related open issue in Gradle to make it fixed before 7.5 is released?

Reply:
It was not a problem with Gradle, just with settings `org.gradle.java.installations.fromEnv` to handle the  JDK18 env variable... I added a comment in CI configuration to remember about that.